### PR TITLE
feat(appeals): add new option to land ownership (owns none of the land)

### PR DESCRIPTION
**Pull request by AlexMoulin**
## Describe your changes
Add 3rd option to site ownership (Not owned by appellant)
Cleaned up naming to align request and response data

## Issue ticket number and link
https://pins-ds.atlassian.net/browse/BOAT-1389
## Type of change 🧩

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update
- [ ] Other (please explain in the description section above)

## Checklist before requesting a review

- [x] I have performed a self-review of my own code
- [x] I have double checked this work does not include any hardcoded secrets or passwords
- [x] I have made corresponding changes to the documentation
- [x] I have provided details on how I have tested my code
- [x] I have referenced the ticket number above
- [x] I have provided a description of the ticket
- [x] I have included unit tests to cover any testable code changes

### CODE REVIEW

File: appeals/web/src/server/appeals/appeal-details/site-ownership/site-ownership.mapper.js [hunk 1]
```diff
@@ -12,7 +12,8 @@ import { appealShortReference } from '#lib/appeals-formatter.js';
 export const changeSiteOwnershipPage = (appealData, appellantCaseData, storedSessionData) => {
 	const shortAppealReference = appealShortReference(appealData.appealReference);
 
-	let isFullyOwned = appellantCaseData.siteOwnership.isFullyOwned;
+	let isFullyOwned = appellantCaseData.siteOwnership.ownsAllLand;
```
Comment (james-hill-kc) [inline appeals/web/src/server/appeals/appeal-details/site-ownership/site-ownership.mapper.js:15]: It doesn't look like these are being reassigned so they could be `const`

File: appeals/web/src/server/lib/mappers/appellantCase.mapper.js [hunk 2]
```diff
@@ -387,7 +387,11 @@ export function initialiseAndMapData(appellantCaseData, appealDetails, currentRo
 					text: 'Site ownership'
 				},
 				value: {
-					text: appellantCaseData.siteOwnership.isFullyOwned ? 'Fully owned' : 'Partially owned'
+					text: appellantCaseData.siteOwnership.ownsAllLand
```
Comment (james-hill-kc) [inline appeals/web/src/server/lib/mappers/appellantCase.mapper.js:390]: the nested ternary is a little hard to read here - brackets or indentation could make it clearer

**Review by james-hill-kc — COMMENTED** [review]
Looks good to me, just a couple of very minor points

File: appeals/web/src/server/appeals/appeal-details/site-ownership/site-ownership.mapper.js (hunk 1 above)
Comment (kandcdev) [reply appeals/web/src/server/appeals/appeal-details/site-ownership/site-ownership.mapper.js:15]: +1

Comment (AlexMoulin) [reply appeals/web/src/server/appeals/appeal-details/site-ownership/site-ownership.mapper.js:15]: Lines 19 and 20 if there is data in the session

Comment (james-hill-kc) [reply appeals/web/src/server/appeals/appeal-details/site-ownership/site-ownership.mapper.js:15]: I can't see `isPrtiallyOwned` being reassigned, but it's really a trivial issue, to happy to resolve this point

**Review by james-hill-kc — APPROVED** [review]
LGTM!

**Pull request merged**